docs/source/introduction.rst

```diff
@@ -0,0 +1,16 @@
+Introduction
+============
+
+**This is a living guide for trustbloc**. If you’d like to contribute, `fork us on GitHub!`_.
+
+What is TrustBloc?
+^^^^^^^^^^^^^^^^^^
+SecureKey Technologies is launching an interoperability initiative aimed at establishing common standards and development frameworks for next generation digital identity networks.
```

Comment:
"SecureKey Technologies is launching" is part of the announcement overview doc, but wouldn't be part of the page.